docs_src/microservices/security/edgex-config-proxy.1.md

```diff
@@ -0,0 +1,127 @@
+% edgex-config-proxy(1) User Manuals edgex-config-proxy(1)
+
+# NAME
+
+**edgex-config-proxy** – Configure EdgeX proxy service
+
+# SYNOPSIS
+
+**edgex-config** proxy [OPTIONS] SUBCOMMAND [ARG...]
+
+# DESCRIPTION
+
+**edgex-config-proxy** configures the EdgeX proxy service.
+
+This command is used to configure the TLS certificate for external connections, create authentication tokens for inbound proxy access, and relevant utility functions.
+
+These commands require access to the secret store master key in order to generate temporary secret store access credentials.
+
+# OPTIONS
+
+  * **-m** _/path/to/resp-init.json_
+
+    Points to location of secret store master key. Defaults to /vault/config/assets/resp-init.json
+
+# COMMANDS
+
+  * **tls**
+
+    Configure inbound TLS certificate. This command will configure the TLS secrets into the secret store and re-deploy them to Kong. Requires additional arguments:
+
+    * --certificate _/path/to/certchain.pem_
+
+      Path to TLS leaf certificate (with optional appended certificate chain, not including a root certificate).
+
+    * --key _/path/to/private.pem_
+
+      Path to TLS private key.
+
+  * **adduser**
+
+    Create a proxy user using JWT public/private key authentication. Requires additional arguments:
+
+    * --jwt (required)
+
+      Create user using JWT authentication plugin.
+
+    * --algorithm HS256 | ES256 (required)
+
+      Algorithm used for signing the JWT.
+
+    * --public\_key _public\_key.pem_ (required)
+
+      Public key used to validate the JWT. This key is assumed to have been pre-created using some external mechanism such as a TPM, HSM, openssl, or other method.
+
+    * --user _username_ (required)
+
+      Username of the user to add.
+
+    * --id _key_
+
+      Optional user-specified &quot;key&quot; used for linkage with an incoming JWT via Kong&#39;s config.key\_claim\_name setting (defaults to &quot;iss&quot; field).
+
+    * --group _group_
+
+      Group to which the user belongs, defaults to &quot;admin&quot;.
+
+    * --output-key _path_
+
+      Output auto-genreated to a file (or &quot;-&quot; for stdout).
+
+  * **deluser**
+
+    Delete a proxy user. Requires additional arguments:
+
+    * --jwt (required)
+
+      Create user using JWT authentication plugin.
```

Comment:
Create => Delete.  and does it use the JWT authentication plugin for delete?

Reply:
Yes, should be delete and that would be done through the chosen auth plugin.